test/functional/test_site_convert_expected/_Sidebar.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```

Comment:
@nicholaschuayunzhi I find it quite strange that `_Sidebar.md` (and `_Footer.md`) is getting rendered here, but I understand that we don't really have a way to stop MarkBind from rendering certain files. Your thoughts?

Reply:
Thanks for pointing that out @yamgent . If users would want to prevent rendering of `_Sidebar.md`, they would have to specify their own addressable pages. 

That being said, I can understand why it would it be confusing and annoying to users to see additional html files that do not make sense (eg Sidebar with header and footer). I'm thinking we may need a page ignore mechanism to prevent its rendering as an addressable page which should be out of scope for this PR. Maybe the files prefixed with `_` should not be addressable?

Alternatives would be to remove the said files as they are no longer used by the site but I'm afraid of tampering with the author's original files as he may want to reuse them/refer to them locally. Potentially in the future, we could store all the original unused files into some location maybe `_original/` and ignore the entire folder for finding addressable pages.

Reply:
Thanks for your response. In view of the limited time, let's consider such issues in a future PR.